libs/angular/spinner/src/spinner.component.ts

```diff
@@ -0,0 +1,10 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'kirby-spinner',
+  template: '<ng-content></ng-content>',
+  standalone: true,
+  imports: [CommonModule],
+})
+export class KirbySpinnerComponent {}
```

Comment:
Do we need to detach ChangeDetectorRef, even though we don't have any attributes to proxy?